src/migrations/20230119000000-create-groups.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable max-len */
+module.exports = {
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(async (transaction) => {
+    const loggedUser = '0';
+    const sessionSig = __filename;
+    const auditDescriptor = 'RUN MIGRATIONS';
+    await queryInterface.sequelize.query(
+      `SELECT
+          set_config('audit.loggedUser', '${loggedUser}', TRUE) as "loggedUser",
+          set_config('audit.transactionId', NULL, TRUE) as "transactionId",
+          set_config('audit.sessionSig', '${sessionSig}', TRUE) as "sessionSig",
+          set_config('audit.auditDescriptor', '${auditDescriptor}', TRUE) as "auditDescriptor";`,
+      { transaction },
+    );
+
+    await queryInterface.createTable('Groups', {
```

Comment:
Can we give anymore context to this table name?

Reply:
UserGrantGroups or something like that.

Reply:
To start groups are a single user, but this will expand to multi-user so I think groups is the correct name. as curated template management will likely leverage the groups